sci-biology/spm/spm-8.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+DESCRIPTION="Analysis of brain imaging data sequences for Octave or Matlab"
+HOMEPAGE="http://www.fil.ion.ucl.ac.uk/spm/"
+SRC_URI="http://www.fil.ion.ucl.ac.uk/spm/download/restricted/idyll/${PN}${PV}.zip"
+
+LICENSE="GPL-2+"
+SLOT="8"
+KEYWORDS="~amd64"
+
+RDEPEND=">=sci-mathematics/octave-3.6.4
+	"
+
+DEPEND="${RDEPEND}
+	app-arch/unzip
+	"
+
+S=${WORKDIR}/${PN}${PV}
+
+src_prepare() {
+	echo "MEXOPTS += -v" >> src/Makefile.var
+	emake -C src distclean PLATFORM=octave
+	emake -C src toolbox-distclean PLATFORM=octave
+}
+
+src_compile() {
+	emake -C src -j1 PLATFORM=octave
```

Comment:
how would I test that? I remember back one and a half years ago when I started to work on this, it failed because some parts were compiled ahead of other stuff they needed. Since the upstream code ha not changed afaict, this should have stayed the same.